src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -116,19 +115,30 @@ public YugabyteDBStreamingChangeEventSource(YugabyteDBConnectorConfig connectorC
 
     @Override
     public void execute(ChangeEventSourceContext context, YugabyteDBPartition partition, YugabyteDBOffsetContext offsetContext) {
-        if (!snapshotter.shouldStream()) {
-            LOGGER.info("Streaming is not enabled in correct configuration");
-            return;
-        }
-
         // replication slot could exist at the time of starting Debezium so
```

Comment:
This comment will also be confusing because it talks about how the Debezium PG connector operates with replication slots which is not true for us, so maybe we can completely remove it or add YB related info here.